R/04-normalize.R

```diff
@@ -120,28 +120,28 @@ gimap_normalize <- function(.data = NULL,
     dplyr::filter(norm_ctrl_flag == "negative_control") %>%
     dplyr::select(pg_ids, dplyr::starts_with("late"))
 
-  # TODO: There's one main median that's found by taking the median of the medians?
-  neg_control_median <- median(apply(neg_control_median_df[, -1], 1, median))
+  # TODO: They find a median for each rep, so apply across columns
```

Comment:
Ah thanks for clarifying this. I couldn't quite determine what kind of median was being calculated.